lib/mix_audit/cli/audit.ex

```diff
@@ -45,4 +47,12 @@ defmodule MixAudit.CLI.Audit do
     |> String.split(",")
     |> Enum.map(&String.trim/1)
   end
+
+  defp ignored_ids_from_file(opts) do
+    opts
+    |> Keyword.get(:ignore_file, ".mix-audit-skips")
+    |> File.read!()
```

Comment:
If the user doesn’t specify the `--ignore-file` option, it will still try to read the `.mix-audit-skips` file. But if it doesn’t exist, it will raise an error?

Reply:
Maybe should read from file only if the option if specified...

Reply:
I copied `bundle audit` system.
the file will be evaluated only if the option is passed

see [a92312e](https://github.com/mirego/mix_audit/pull/27/commits/a92312ef2d1735432580881df53672b8a3c0c407)